app/src/docker/entrypoint.sh

```diff
@@ -3,6 +3,6 @@
 
 eval exec java -jar \
   -Djava.security.egd=file:/dev/./urandom \
-  $JAVA_PROFILE \
-  $JAVA_OPTS \
+  "$JAVA_PROFILE" \
+  "$JAVA_OPTS" \
```

Comment:
Undo this change b/c `$JAVA_OPTS` may have space delimited arguments intended to be treated as separate arguments. Adding quotes causes the entire `$JAVA_OPTS` value to be treated as a single argument. The `eval exec` may help prevent that but testing is required.